### PR TITLE
bump to 5.0.0-beta

### PR DESCRIPTION
**Pull request by gwenshap**
Plus a tweak to get C3 to work with KSQL

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -372,4 +373,4 @@ services:
       KSQL_OPTS: -Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
                   -Djavax.net.ssl.trustStorePassword=confluent
                   -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.client.keystore.jks
-                  -Djavax.net.ssl.keyStorePassword=confluent
+                  -Djavax.net.ssl.keyStorePassword=confluent -Dhost.name=ksql-cli
```
Comment (ybyzek) [inline docker-compose.yml]: @gwenshap , would setting `KSQL_HOST_NAME=ksql-cli` work instead?  For consistency...

Comment (gwenshap) [reply docker-compose.yml]: I can try.

**Comment by ybyzek** [comment]
@gwenshap : wonderful addition to cp-demo, I have tested it and love the new features!  Thank you.

**Comment by ybyzek** [comment]
I reverted the initial PR which committed to master.  This change is now in a new `5.0.0-beta` branch

**Pull request merged**